src/backend/codegen/codegen_interface.cc

```diff
@@ -0,0 +1,21 @@
+//---------------------------------------------------------------------------
+//  Greenplum Database
+//  Copyright (C) 2016 Pivotal Software, Inc.
+//
+//  @filename:
+//    codegen_interface.cc
+//
+//  @doc:
+//    Implementation of codegen interface's static function
+//
+//---------------------------------------------------------------------------
+#include "codegen/codegen_interface.h"
+
+using gpcodegen::CodeGenInterface;
+
+unsigned CodeGenInterface::unique_counter_ = 0;
```

Comment:
Comments for this variable is in header file. Do you still see a need to add here ?